user_docs/en/changes.t2t

```diff
@@ -7,7 +7,9 @@ What's New in NVDA
 
 == New Features ==
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
--
+- Add-on Store:
+  - The minimal version and the latest tested version of NVDA for an extension appear in the "Other Information" field of an extension
+

```

Comment:
1. In t2t files, a dash with nothing else is used to close a list. Do not remove it please
2. In French we use the word "extension", but in English it's "add-on".
3. No need to repeat the word "add-on"; once is enough.
4. appear -> appears
5. The sentence should be finished with a period.
6. I do not think that you should create an Add-on Store group for now in the change log; this may be done in the future if there are various changes on this topic.
7. You should add the number of the issue that you are fixing as a reference.
8. (optional) You may want to add your GitHub's name to indicate that you are the author of this contribution. It's not mandatory, so it's up to you; my personal opinion is that it's a plus to have the contributors listed in the change log, but do as you want.

```suggestion
- In the Add-on Store, the minimal version and the latest tested version of NVDA for an add-on now appears in the "Other Information" field. (#15776, @Nael-Sayegh Nael-Sayegh )
- 
```